### ISSUE TITLE
Update changelog

### ISSUE THREAD

**Issue by hypesystem** [ToothlessGear/node-gcm#322]
The changelog has fallen behind as of recent. We should update it to make sure it reflects which changes were introduced in which versions.

**Comment by eladnava** [comment]
Seems like this is partially addressed via https://github.com/ToothlessGear/node-gcm/pull/319/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e.

**Comment by hypesystem** [comment]
Yes! That's what made me realize we had forgotten to do this for a while. But we need a bit more detail, and to make sure that we include all the versions we have missed.

**Comment by Schweinepriester** [comment]
Updating the changelog would be helpful :)

**Comment by mdesson** [comment]
Are there any known breaking changes within `v1.x.x`? I know this project uses semver, but I usually read through the changelog of a given package to validate before going through an upgrade. An updated changelog would be a great help
